ynr/apps/popolo/api/next/serializers.py

```diff
@@ -50,7 +50,7 @@ class Meta:
         fields = ("elected", "num_ballots")
         ref_name = None  # Tells swagger that this is always embedded
 
-    elected = serializers.ReadOnlyField(source="is_winner")
+    elected = serializers.ReadOnlyField(source="membership.elected")
```

Comment:
I'm fairly sure this is ok, but can you confirm that this doesn't cause O(n) queries? I think we're already using `select_related` or `prefetch_related` for Memberships

Reply:
Yes, we are using [prefetch related on the BallotViewSet](https://github.com/DemocracyClub/yournextrepresentative/blob/master/ynr/apps/elections/api/next/api_views.py#L43) where this serializer is ultimately used. Also checked the query count/speed using debug toolbar and it is the same as on master.